vendor-extra/ViewsBundle/src/Views/InlineViewConverter.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\ViewsBundle\Views;
+
+use FluentDOM\DOM\Node\NonDocumentTypeChildNode;
+
+interface InlineViewConverter
```

Comment:
Wondering whether this should be separate at all. There's three types of things to convert:

1. Chunk of data into a known pattern (eg front matter to content header)
2. List of mixed content into unknown patterns (eg mixed text and inline elements into a variety of patterns)
3. Chunks of data into unknown patterns (eg list of paragraphs, figures etc into a variety of patterns)

Currently 1 and 2 have dedicated interfaces, rather than introduce a third we can merge them all into 1:

```php
interface ViewConverter
{
    public function convert(NonDocumentTypeChildNode $object, ?string $template = null, array $context = []) : View;
}
```

Then the implementation filters out non-elements, like https://github.com/libero/browser/blob/12e7c1d0bb224e386236a958fb6932e10d7646ba/vendor-extra/ViewsBundle/src/Views/InlineViewConverterRegistry.php#L22-L24 (but also checking the template).

Reply:
At some point we should find optimisations for the visitors (eg split them by expected template, element).

Reply:
I've pushed this change in bca13a4, still open for discussion.